content/en/security/detection_rules/_index.md

```diff
@@ -7,112 +7,133 @@ aliases:
 
-- [Cloud SIEM][2] uses log detection to analyze ingested logs in real-time. You can also create [custom detection rules][3] to tailor to your environment.
-
-- [Cloud Security Management Misconfigurations][4] uses cloud configuration and infrastructure configuration detection rules to scan the state of your cloud environment.
-
-- With [Cloud Security Management Threats][5], the Datadog Agent actively monitors system activity and evaluates it against a set of detection rules.
+Out-of-the box rules are available for the following security products:
 
+- [Cloud SIEM][2] uses log detection to analyze ingested logs in real-time.
+- Cloud Security Management (CSM):
+    - [CSM Misconfigurations][4] uses cloud configuration and infrastructure configuration detection rules to scan the state of your cloud environment.
+    - [CSM Threats][5] uses the Datadog Agent and detection rules to actively monitor and evaluate system activity.
+    - [CSM Identity Risks][14] uses detection rules to detect IAM-based risks in your cloud infrastructure.
 - [Application Security Management][6] (ASM) leverages Datadog [APM][7], the [Datadog Agent][8], and detection rules to detect threats in your application environment.
 
-## Creating and managing detection rules
+## Custom detection rules
+
+There may be situations where you need to customize a rule based on your environment or workload. For example, if you're using ASM, you may want to customize a detection rule that detects users performing sensitive actions from a geolocation where your business doesn't operate.
 
-The [Detection Rules][9] page lets you search all detection rules by rule type. Quickly enable, disable, edit, delete, and clone rules. To create a custom [detection rule][3], click on the **New Rule** button in the top right corner of the page.
+To [create custom rules](#create-detection-rules), you can clone the default rules and edit the copies, or create your own rules from scratch.
 
-### Finding detection rules
+## Search and filter detection rules
 
-The free text search filters Detection Rules by text in the rule name or query. Query results update in real-time when the query is edited—there is no "Search" button to click.
+To view out-of-the-box and custom detection rules in Datadog, navigate to the [**Security** > **Configuration**][15] page. Rules are listed on separate pages for each product (Application Security, Cloud Security Management, and Cloud SIEM).
 
-#### Filter by facet
+To search and filter the rules, use the search box and facets to query by value. For example, if you have several rule types, such as `log detection` or `cloud configuration`, filter by `only` to show only rules for the selected rule type. You can also filter by facets such as `source` and `severity` when investigating and triaging incoming issues.
```

Comment:
```suggestion
To search and filter the rules, use the search box and facets to query by value. For example, if you have several rule types, such as `log detection` or `cloud configuration`, filter by `only` to show only rules for the selected rule type. You can also filter by facets such as `source` and `severity` when investigating and triaging incoming issues.
```

I am a little confused on this sentence, are you trying to say "filer to show specific rules for the selected rule type". Filter by `only` doesn't exactly make sense, as the way it reads, I am looking for a filter option called `only`.  So I think we can re-word that a bit to be more clear.

Reply:
In this case, there is a filter option called only that shows in the UI when you hover over a rule type. I've updated the verbiage to make that clearer.